code/modules/projectiles/guns/projectile/launchers.dm

```diff
@@ -30,7 +30,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/gyropistol
-	name = "gyrojet pistol"
+	name = "\improper Gyrojet pistol"
```

Comment:
I think this should be "rocket pistol" because just saying "pistol" doesn't quite do the gyrojet justice